### PR TITLE
pilgrim un-tuning part 1/2

### PR DESCRIPTION
**Pull request by Hmeister-real**
## About The Pull Request

They were too overtuned because they were made with the intent to be an overflow to adventurers originally. This makes them less combat viable, and adjusts them for the XP changes that are upcoming. Rather than complain about "boohoo, boohoo, i cant learn carpentry+masonry+Mining+lumberjacking+whatever the fuck else" now pilgrims are specialised + a little extra, at the cost of being weaker since we have more than enough combat viable roles, and also Lumberjack, miner and carpenter were literally just adventurers lol

## Why It's Good For The Game

 balance is good. Draft because i need outside opinions

### CODE REVIEW

**Comment by Teragenic** [comment]
I have some feedback about this PR. 

Drunkard doesn't need a nerf. It's already quite bad role already. 
Miner does not need a speed nerf. Most racials already handle speed nerf. Dwarven miners would get a -4(?) or 5. Even with a certain PR that seems too much.
Hunters are also still expected to be out in the wilderness. I'd suggest making them atleast keep their axe skill. It's literally Novice.
Seamstress seems overtuned. It doesn't already account for racials. At best you can get +5 intelligence or -6 strength. No need to edit the stats there.
Naked and afraid also being tweaked in stats is also weird.


All in all. I feel this PR is poorly tweaked in stats and skills. While good in some parts. It definitely does not account for the races in particular or actual balance of some classes.

File: code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/blacksmith.dm [hunk 1]
```diff
@@ -33,23 +33,22 @@
...
+		H.mind.adjust_skillrank(/datum/skill/combat/knives, 2, TRUE) 
 		H.mind.adjust_skillrank(/datum/skill/craft/crafting, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/craft/masonry, 1, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/craft/engineering, 2, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/reading, 1, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/craft/blacksmithing, 2, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/craft/armorsmithing, 2, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/craft/weaponsmithing, 2, TRUE)
 		H.change_stat("strength", 1)
-		H.change_stat("endurance", 2)
```
[the remark this answers is not in the document]
Comment (Hmeister-real) [reply code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/blacksmith.dm:52]: added back end

File: code/modules/jobs/job_types/roguetown/adventurer/types/special/nakedandafraid.dm [hunk 2]
```diff
@@ -16,13 +16,17 @@
...
+		H.mind.adjust_skillrank(/datum/skill/labor/butchering, 2, TRUE)		
 		H.mind.adjust_skillrank(/datum/skill/misc/climbing, 4, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/swimming, 4, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/medicine, 2, TRUE)
-	H.change_stat("strength", 2)
+	H.change_stat("strength", 3)
 	H.change_stat("endurance", 4)
-	H.change_stat("constitution", 1)
-	H.change_stat("intelligence", -3)
+	H.change_stat("constitution", 3)
```
[the remark this answers is not in the document]
Comment (Hmeister-real) [reply code/modules/jobs/job_types/roguetown/adventurer/types/special/nakedandafraid.dm]: it is now just 2 con, so its a +1 for -2 int

**Comment by Hmeister-real** [comment]
> Oh also, why is Minstrel getting a pickpocket increase. We had/have people already abuse the system by stealing from oneself.

Because new XP stops that, and there needs to be more crime

**Comment by Hmeister-real** [comment]
> -5 int for +3 CON is not a fair trade, its extremely beneficial for the person taking it.

Its now -5 int for 2 con. Before it was -3 int for 1 con

**Comment by Stagrawl** [comment]
I'll point out that gambler is supposed to be an RP intensive role with extra utilities that haven't been coded in **yet**

**Comment by Hmeister-real** [comment]
> I'll point out that gambler is supposed to be an RP intensive role with extra utilities that haven't been coded in **yet** and while I'm not wholly opposed to buffs, they need to keep this broadly in mind.

Oh yeah i know, thats why i just removed the dead trait to not confuse people, and gave them an unarmed+wrestle buff. i remember that being discussed before

**Comment by Stagrawl** [comment]
All good on that front, not going to comment on this PR further than offering some clarifications as to the intent behind some roles really.

**Comment by Hmeister-real** [comment]
im gonna put it as ready for review because, im gonna make a 2/2 where i make changes to adventurer. And also imo these changes are good

**Comment by Azlier** [comment]
The obsession with stripping away polearms from everyone is very odd considering it governs the use of staffs, THE quintessential traveler weapon both historically and in-game.

**Comment by TheFenix323** [comment]
Negative speed for miners is rough, since how fast they swing the pickaxe is determined by speed and is worse for dwarfs who get -3/-2 already

**Comment by Gr33d-y** [comment]
Not much of a fan of this one.

**Comment by dotCatshark** [comment]
Why increase the nudist intelligence debuff to -5? It's already needlessly high as is. Also they definitely do not need whips. And why lower their crafting skill?

**Comment by Hmeister-real** [comment]
> Why increase the nudist intelligence debuff to -5? It's already needlessly high as is. Also they definitely do not need whips. And why lower their crafting skill? Also this nerfs a lot of classes. Why?

It doesn't lower the craft skill? And it nerfs pilgrim to be as they should since they were overtuned to compete with Blackstones frag first mentality, here they should be weak, they aren't combat roles

**Pull request closed without merging**